File: test-suite-docs-java/src/test/java/io/micronaut/mcp/docs/tools/jsonschema/FenEvaluationRequest.java
package io.micronaut.mcp.docs.tools.jsonschema;

//tag::imports[]
import io.micronaut.jsonschema.JsonSchema;
import io.micronaut.serde.annotation.Serdeable;

//end::imports[]
//tag::clazz[]
/**
 *
 * @param fen A Chess position in Forsyth–Edwards Notation
 */
@JsonSchema
@Serdeable
public record FenEvaluationRequest(String fen) {
}
//end::clazz[]

File: test-suite-docs-kotlin/src/test/kotlin/io/micronaut/mcp/docs/tools/jsonschema/FenEvaluationRequest.kt
package io.micronaut.mcp.docs.tools.jsonschema

//tag::imports[]
import io.micronaut.jsonschema.JsonSchema
import io.micronaut.serde.annotation.Serdeable

//end::imports[]
//tag::clazz[]
@JsonSchema
@Serdeable
data class FenEvaluationRequest(
    /** A Chess position in Forsyth–Edwards Notation */
    val fen: String
)
//end::clazz[]
